code-examples/protect-page-login/flutter_web_redirect/env

```diff
@@ -1 +1 @@
-ORY_BASE_URL=http://localhost:3005
+ORY_BASE_URL=http://localhost:3005/.ory
```

Comment:
revert please!

Reply:
ok thanks. I'll also update the documentation as it says to use Ory Proxy.

Reply:
Thank you :)